content/applications/marketing/marketing_automation/getting_started/target_audience.rst

```diff
@@ -1,123 +1,251 @@
-==================
-Target an audience
-==================
+=======================
+Target audience filters
+=======================
+
+When trying to grow a business, it is paramount for companies to send marketing campaigns to the
+*right* audience.
 
-Delivering marketing campaigns to the right audience is paramount when trying to grow a business.
 The Odoo *Marketing Automation* application helps marketers to do just that by providing detailed
-filtering tools, which can be as simple (or as complex) as necessary, to reach the right customers
-at the right time.
+target audience filtering tools, which can be as simple or as complex as necessary, to reach the
+right customers at the right time.
 
 Configure target filters
 ========================
 
-When configuring the target filters on a marketing campaign, there are some options that have an
-:guilabel:`> (arrow)` icon beside them. The :guilabel:`> (arrow)` icon signifies that the particular
-filter has more refined parameters within it that can be customized.
+The following is working off basic marketing automation campaign information covered in the
+:doc:`marketing automation campaigns documentation <first_campaign>`.
```

Comment:
Would move this above **Configure target filters** and format as a **Note** admonition.